zerver/models/realms.py

```diff
@@ -768,6 +771,21 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
                 SystemGroups.NOBODY,
             ],
         ),
+        can_create_groups=GroupPermissionSetting(
+            require_system_group=True,
```

Comment:
I guess we can set `require_system_group` as `False` like we are doing for other new settings as we do not allow to set the setting to non-system groups in production anyways.

Reply:
fixed!